wp-content/mu-plugins/pantheon/pantheon-login-form-mods.php

```diff
@@ -3,53 +3,72 @@
  * Modify the WordPress login form for Pantheon
  */
 
-/**
- * Enqueue Pantheon login styles
- *
- * @return void
- */
-function Pantheon_Enqueue_Login_style()
-{
-    wp_enqueue_style('pantheon-login-mods', plugin_dir_url(__FILE__) . 'assets/css/return-to-pantheon-button.css', false); 
-}
+ /**
+  * Should we proceed with adding the
+  * return to Pantheon button?
+  *
+  * Only if:
+  * "RETURN_TO_PANTHEON_BUTTON" is not false,
+  * we are not on the test or live environments
+  * and we are on a Pantheon subdomain
+  */
+$show_return_to_pantheon_button = apply_filters( 'show_return_to_pantheon_button', ( 
+    ( ! defined('RETURN_TO_PANTHEON_BUTTON') || RETURN_TO_PANTHEON_BUTTON ) &&
+    ! in_array($_ENV['PANTHEON_ENVIRONMENT'], array( 'test', 'live' ) ) &&
+    false !== stripos( get_site_url(), 'pantheonsite.io')
```

Comment:
@ari-gold I am using [`get_site_url`](https://developer.wordpress.org/reference/functions/get_site_url/) to check for `pantheonsite.io`. This pulls the domain value from WordPress. Do you think we should also check `$_SERVER['HTTP_HOST']`?